### PR TITLE
Code Fixes For Action.Submit Button - Bot Adaptive Card Actions

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-adaptive-card-actions/nodejs/bots/adaptivecardActionsBot.js [hunk 1]
```diff
@@ -16,35 +16,37 @@ class AdaptiveCardActionsBot extends ActivityHandler {
...
-            else if (text.includes("Red") || text.includes("Blue") || text.includes("Yellow")) {
-                // Create an array with the valid color options.
-                const validColors = ['Red', 'Blue', 'Yellow'];
-
-                // If the `text` is in the Array, a valid color was selected and send agreement.
-                if (validColors.includes(text)) {
-                    await context.sendActivity(`I agree, ${text} is the best color.`);
+            if (context.activity.text != null) {
+                const text = context.activity.text;
+                if (text.includes("Card Actions")) {
```
Comment (ChetanSharma-msft) [inline samples/bot-adaptive-card-actions/nodejs/bots/adaptivecardActionsBot.js:22]: Put one line space.

File: samples/bot-adaptive-card-actions/nodejs/bots/adaptivecardActionsBot.js [hunk 2]
```diff
@@ -16,35 +16,37 @@ class AdaptiveCardActionsBot extends ActivityHandler {
...
+                }
+                else if (text.includes("Red") || text.includes("Blue") || text.includes("Yellow")) {
+                    // Create an array with the valid color options.
+                    const validColors = ['Red', 'Blue', 'Yellow'];
+
+                    // If the `text` is in the Array, a valid color was selected and send agreement.
+                    if (validColors.includes(text)) {
+                        await context.sendActivity(`I agree, ${text} is the best color.`);
+                    }
+                    await this.sendSuggestedActions(context);
```
Comment (ChetanSharma-msft) [inline samples/bot-adaptive-card-actions/nodejs/bots/adaptivecardActionsBot.js:38]: Put one line space.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added NIT comments.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by ChetanSharma-msft** [comment]
@Mohammed-MSFT - Please run build pipeline.

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Pull request merged**